lms/djangoapps/courseware/courses.py

```diff
@@ -298,7 +313,32 @@ def get_course_info_section(request, course, section_key):
     html = ''
     if info_module is not None:
         try:
-            html = info_module.render(STUDENT_VIEW).content
+           # html = info_module.render(STUDENT_VIEW).content
+            html_pre = info_module.render(STUDENT_VIEW).content
+            root = lxml.html.fromstring(html_pre)
+            for article in root.findall('article'):
+                article_date = article.find('h2').text
+                # Updates later than now  are not shown.
+                try:
+                    if parser.parse(article_date) > datetime.datetime.now():
+                        root.remove(article)
+                except ValueError:
+                    pass
+            # Updates are shown with user locale format. Updates without date are not affected.
+            for article in root.findall('article'):
+                for heading in article.iter('h2'):
+                    try:
+                        heading.text = formats.date_format(parser.parse(heading.text))#"SHORT_DATE_FORMAT") 
+                    except ValueError:
+                        heading.text = heading.text
+
+
```

Comment:
You have too many spaces in your code which is causing linting violations. Please see https://github.com/edx/edx-platform/blob/master/docs/en_us/internal/testing.rst#python-code-style-quality for instructions on how to run linting locally